pkg/storage/mvcc_test.go

```diff
@@ -5248,6 +5248,58 @@ func TestResolveIntentWithLowerEpoch(t *testing.T) {
 	}
 }
 
+// TestTimeSeriesMVCCStats ensures that merge operations
+// result in expected increase in timeseries data
+func TestTimeSeriesMVCCStats(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	defer log.Scope(t).Close(t)
+
+	ctx := context.Background()
+	for _, engineImpl := range mvccEngineImpls {
+		t.Run(engineImpl.name, func(t *testing.T) {
+			engine := engineImpl.create()
+			defer engine.Close()
+			var ms = enginepb.MVCCStats{}
+
+			// Perform the same sequence of merges on the same key
+			// twice and record timeseries data
```

Comment:
[nit] Perform a sequence of merges on the same key and record the MVCC stats for it.